kolibri/plugins/coach/assets/src/views/common/notifications/ActivityList.vue

```diff
@@ -189,7 +189,11 @@
             this.notifications = [
               ...this.notifications,
               ...data.results.map(this.reshapeNotification).filter(Boolean),
```

Comment:
This should do it. You can probably also filter them out of `data.results` to save us the step of having to map `reshapeNotification` over them.

Reply:
Thanks @jonboiser ! Fixed in latest commit